tests/concurrent-restore/run.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+#
+# Copyright 2019 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eu
+
+# Populate the mydumper source
+DBPATH="$TEST_DIR/restore.mydump"
```

Comment:
Please rename the folder to something other than `restore.mydump` (nor `cppk.mydump` nor `cpch.mydump`).